akka-stream/src/main/scala/akka/stream/io/IOResult.scala

```diff
@@ -0,0 +1,33 @@
+package akka.stream.io
+
+import akka.Done
+import scala.util.{ Failure, Success, Try }
+
+/**
+ * Holds a result of an IO operation.
+ *
+ * @param count A number of IO operations performed.
```

Comment:
What about something like "Numeric value depending on context, for example IO operations performed or Bytes processed"?